src/firmware/app/primitives/move_manager.h

```diff
@@ -0,0 +1,23 @@
+#pragma once
+
+#include <stdbool.h>
+
+#include "firmware/app/world/firmware_world.h"
+#include "shared/proto/primitive.nanopb.h"
+
+/**
+ * Start moving in the given world
+ * @param void_state_ptr [in] A pointer to this primitives state, as allocated by the
+ *                            `create_state` function
+ * @param world [in] The world this primitive is running in.
+ * @param move_position_params Move Position Params
+ * @param final_angle The final angle
+ */
+void app_move_manager_start(void *void_state_ptr, FirmwareWorld_t *world,
+                            MovePositionParams move_position_params, float final_angle);
```

Comment:
Why not call this `move_primitive` instead? It effectively acts as one, no?

Reply:
yes but it only handles the movement part of the move_primitive, not the dribbling

Reply:
Sorry, still a bit confused here. Seems like semantically this could just be "move without dribbling or autokicking or autochipping" and then the other primitives could just build off this one?

Reply:
Gonna run into the same issue as https://github.com/UBC-Thunderbots/Software/pull/1645#discussion_r464014504 so I don't think it's worth it